### DISCUSSION TITLE
Question: Package version numbering change

### DISCUSSION THREAD

**Discussion by robocoder** [oerdnj/deb.sury.org#1798] [Q&A]
The package version numbering scheme has changed for php 7.4 and php 8.0. There's a `1:` prefix. Is this the new convention going forward?

| Package | Version | Uploaded by |
|----|----|----|
| php7.4 | 1:7.4.30-1+ubuntu22.04.1+deb.sury.org+1 | Ondřej Surý (2022-06-10) |
| php7.4 | 1:7.4.30-1+ubuntu21.10.1+deb.sury.org+1 | Ondřej Surý (2022-06-10) |
| php7.4 | 1:7.4.30-1+ubuntu20.04.1+deb.sury.org+1 | Ondřej Surý (2022-06-10) |
| php7.4 | 1:7.4.30-1+ubuntu18.04.1+deb.sury.org+1 | Ondřej Surý (2022-06-10) |
| php8.0 | 1:8.0.20-1+ubuntu22.04.1+deb.sury.org+1 | Ondřej Surý (2022-06-10) |
| php8.0 | 1:8.0.20-1+ubuntu21.10.1+deb.sury.org+1 | Ondřej Surý (2022-06-10) |
| php8.0 | 1:8.0.20-1+ubuntu20.04.1+deb.sury.org+1 | Ondřej Surý (2022-06-10) |
| php8.0 | 1:8.0.20-1+ubuntu18.04.1+deb.sury.org+1 | Ondřej Surý (2022-06-10) |

**Comment by oerdnj** [comment]
Ask the lovely folks at Canonical who did this abomination: https://packages.ubuntu.com/jammy/libapache2-mod-php7.4 and broke the versioning just for everybody else.

**Reply by robocoder** [reply]
Thanks for the explanation.
